docs/admin/init-containers.rst

```diff
@@ -10,8 +10,8 @@ Write an init container
 
 Nublado can run any containers of your choice as init containers.
 
-Only the default entry point will be run.
-There is no way to override the default entry point with a custom command.
+The container command can be specified as a comma-separated list of strings, such as ``["nublado", "inithome"]``.
```

Comment:
It's a YAML list, right? So that's just one syntax that can be used; you can also put each element on a separate line starting with `-`. I don't think you need to describe YAML syntax; you can just say it's a YAML list.